docs/app/pages/components/components-sections/wizard/wizard/wizard.component.html

```diff
@@ -1,6 +1,6 @@
 <ux-wizard [orientation]="orientation" (stepChange)="onStepChange($event)">
 
-    <ux-wizard-step *ngFor="let step of steps" [header]="step.header">
+    <ux-wizard-step *ngFor="let step of steps" [header]="step.header" [validator]="userValidator">
```

Comment:
I think we discussed not modifying the documentation example, so let's drop this change.